examples/material-ui-remix-ts/app/root.tsx

```diff
@@ -1,128 +1,43 @@
-import * as React from 'react';
 import {
   Links,
-  LiveReload,
   Meta,
   Outlet,
   Scripts,
   ScrollRestoration,
-  useRouteError,
-  isRouteErrorResponse,
-} from '@remix-run/react';
-import { withEmotionCache } from '@emotion/react';
-import { unstable_useEnhancedEffect as useEnhancedEffect } from '@mui/material';
-import theme from './src/theme';
-import ClientStyleContext from './src/ClientStyleContext';
-import Layout from './src/Layout';
+} from "@remix-run/react";
 
-interface DocumentProps {
-  children: React.ReactNode;
-  title?: string;
-}
-
-const Document = withEmotionCache(({ children, title }: DocumentProps, emotionCache) => {
-  const clientStyleData = React.useContext(ClientStyleContext);
+import { getMuiLinks } from "./mui/getMuiLinks";
+import { MuiMeta } from "./mui/MuiMeta";
+import { LinksFunction } from "@remix-run/node";
+import { MuiDocument } from "./mui/MuiDocument";
 
```

Comment:
Why is the `createEmotionCache` function not there anymore?

Reply:
It is something I ask reviewers to review it in depth. because it is not the area of my expertise.  
I explained in the description as well:

> Emotion has changed alot and it seems there is no need to do lots of the changes(i.e. cache + theme management) that had been done previously as it is there out of the box for NextJs and probably for Remix (https://emotion.sh/docs/ssr#nextjs)

TLDR; it seems new version of Emotions handles cahce. Also my tests didn't show any difference.